### PR TITLE
Need to calculate 2 patterns for all java files in parallel

### PR DESCRIPTION
**Pull request by lyriccoder**
https://github.com/yegor256/aibolit/issues/26

Added entropy metrics and fix previous errors with flake8

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#39](https://github.com/yegor256/aibolit/issues/39) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/a3fd7347-b022-4a3b-8f24-874030704a5a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: scripts/04-find-patterns.py [hunk 1]
```diff
@@ -20,73 +20,86 @@
...
-                callback=log_result)
+                callback=log_func)
 
     pool.close()
     pool.join()
 
-    t2 = time.time()
-    print('It took this many seconds ' + str(t2 - t1))
+    end = time.time()
+    print('It took {} seconds'.format(str(end - start)))
```
Comment (carlosmiranda) [inline scripts/04-find-patterns.py]: @lyriccoder we don't need `str` here anymore, `format` will take care of string conversion.

**Comment by 0crat** [comment]
Job was finished in 21 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
Pull request [#39](https://github.com/yegor256/aibolit/issues/39) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Pull request closed without merging**